http-server/src/main/java/io/airlift/http/server/HttpServerConfig.java

```diff
@@ -183,6 +184,18 @@ public HttpServerConfig setKeystorePassword(String keystorePassword)
         return this;
     }
 
+    public String getKeyManagerPassword()
+    {
+        return keyManagerPassword;
+    }
+
+    @Config("http-server.https.keymanager.password")
+    public HttpServerConfig setKeyManagerPassword(String keyManagerPassword)
```

Comment:
We should add `@ConfigSecuritySensitive` so the password isn't printed in the log

Reply:
I didn't know, I saw you added it, thanks for that, electrum!